app/src/main/java/org/jboss/hal/client/skeleton/HeaderView.java

```diff
@@ -272,6 +272,8 @@ public HeaderView(Places places, AccessControl ac, Resources resources) {
         if (!su) {
             topLevelCategories.removeChild(patching);
             topLevelCategories.removeChild(accessControl);
+        } else if (!environment.isPatchingEnabled()) {
```

Comment:
Shouldn't that be an independent if?

Reply:
It can be separate, but I don't think it matters in this case. If the user is not an administrator, patching category will always be removed in first condition, if the user is and admin, it will check if patching is enabled.

If you think it's clearer as separate condition, I'll change it.

Reply:
You're right it should not matter. But I think it's clearer if it's separate.

Reply:
OK, changed now